### DISCUSSION TITLE
在路由器上跑总是自己崩你们都是怎么调试的

### DISCUSSION THREAD

**Discussion by chanhsiaoyu** [XTLS/Xray-core#1047] [Q&A]
asus ax86u 弄了个透明代理上去，总是不定时的几个小时崩了，也没留下什么线索，只好搞个crontab 30查下进程在不在再拉起。。。这种情况你们都是怎么排查原因的，log也看不出什么，怎么能让它崩的时候产生个coredump啥的。

**Comment by chanhsiaoyu** [comment]
那换个问法,经常在路由器上掉出或者假死,你们是怎么写监控脚本的?每分钟curl一次google吗,还是xray本身有没有什么测联通性的命令,比如xray --test 啥的?

**Reply by capxax** [reply]
用armv7 不要v8
